src/components/RightPane/CoursePane/SearchForm/SearchForm.js

```diff
@@ -64,16 +75,27 @@ const SearchForm = (props) => {
                     </div>
 
                     {showLegacySearch && (
-                        <LegacySearch
-                            onSubmit={() => {
-                                logAnalytics({
-                                    category: analyticsEnum.classSearch.title,
-                                    action: analyticsEnum.classSearch.actions.MANUAL_SEARCH,
-                                });
-                                toggleSearch();
-                            }}
-                            onReset={resetFormValues}
-                        />
+                        <Suspense
```

Comment:
Why does this need a suspense? Are there any network calls?

It does look a bit strange having the course search loading gif displayed here temporarily

Reply:
I believe it's good practice to wrap lazy-loaded components in Suspenses, but it might also work without (haven't tested it).

I just decided to use the course search loading gif as a placeholder for the fallback, but we could just leave it blank as well.

Reply:
Oh yeah no you should really only lazy load something if there is a blocker in the component (e.g. an awaited network call). So a suspense should not be used here.

Reply:
Oh I remember why I wanted to lazy load legacy search now. It's because it uses the department search bar, which in turn includes the static department list, which was one of the stretch goals for #403.

Removing the suspense breaks things, but even an empty div in the fallback works, and should be less abrupt than the gif.
![image](https://user-images.githubusercontent.com/89349085/179048599-edb5dccf-a263-41b4-ab71-d0df82679350.png)

Reply:
This is our bundle without lazy loading the map.
![image](https://user-images.githubusercontent.com/48658337/179344112-f3aaeb26-05a2-4c1c-ac48-753315616c03.png)
Looks like the bundle is roughly 60 kb bigger without the lazy loading, so I think this is where most of the performance is coming from.

Reply:
Also, reading the earlier conversation it seems like Chase really meant to ask, "why is this being lazy-loaded", because the response Eddy provided was basically that it needs a suspense is _because_ it is being lazy loaded, but I think that got cleared up. For extra clarification, the Suspense is needed to specify content to display while the lazy-loaded component is being loaded (in my understanding).

Reply:
I think we should just remove the lazy loading for the legacy search, given the negligible performance boost now that I actually did some research. Having our codebase remain as simple as possible is a bigger priority than a 1% performance boost. This probably means removing the lazy loading on the images too, but keeping it on the map since that does provide a significant boost.

Reply:
Now that I'm realizing we should prioratize based on file sizes, here are some other candidates for lazy loading and their uncompressed sizes:
- RightPane/CoursePane/static folder (the 'no results' images) - 70 kb
- SectionTable folder - 70 kb
- RightPane\CoursePane\SearchForm\Gifs - 2 MB
  - Maybe this gets relatively smaller on the bundle because image compression is better than source code compression, but I think this may be a major reason why our bundle is large. For reference, the entire source folder is 2.5 MB, so lazy loading this gets us potentially 80% size reduction at best.